app/helpers/users_helper.rb

```diff
@@ -7,7 +7,7 @@ def roles
 
   def roles_label
     label = User.human_attribute_name 'role'
-    link  = link_to '#users-role-help', data: { toggle: 'modal' } do
+    link  = link_to nil, data: { start_tour: true, steps: [t('tours.users.roles')] } do
```

Comment:
Creo que mejor lo dejamos como `tour`:

```suggestion
    link  = link_to nil, data: { tour: true, steps: [t('tours.users.roles')] } do
```

Reply:
Podemos directamente buscar por `[data-tour-steps]`  y no "gastamos" una key, te parece? jaja

Reply:
O más osado, que `data-tour` tenga dentro los pasos y listo... lo amaste =P

Reply:
Jajaja si.